src/app/modules/card-cluster/card-cluster.component.ts

```diff
@@ -94,6 +94,16 @@ export class AppCardClusterComponent implements OnInit {
       }
     }
     this.resize();
+
+    // Handle optional parameters appropriately
+    if (this.cardArray.cluster.length > 0) {
+      this.cardArray.cluster.forEach(function(card, index) {
```

Comment:
Ahhh, that's right - I also needed to set `idPrefix` before entering the `forEach` b/c it's only on the parent object to the cluster array we're looping over. So running without I get a bunch of errors stating `cardArray` is undefined prop.